validator/client/beacon-api/BUILD.bazel

```diff
@@ -1,15 +1,46 @@
 load("@prysm//tools/go:def.bzl", "go_library")
+load("@io_bazel_rules_go//go:def.bzl", "go_test")
 
 # Gazelle will flag this for removal since it's only being used with the use_beacon_api flag, but it should be kept
 # keep
+# gazelle:build_tag use_beacon_api
```

Comment:
```suggestion
# gazelle:build_tag use_beacon_api
```

You shouldn't need the keep directive if the build_tag directive is working properly.